### PR TITLE
Strings become global regexes.

### PR DESCRIPTION
**Pull request by ljharb**
Fixes #30.

### CODE REVIEW

**Comment by littledan** [comment]
What made you decide against adding `u` as well here?

**Comment by ljharb** [comment]
I still lean towards using it, but I don't think it's as much of an obvious change, and since we're discussing #27 with the committee anyways, I plan to suggest it there.

**Comment by ljharb** [comment]
Committee consensus today was that only the `global` flag would be added, not the `unicode` flag.

File: spec.emu [hunk 1]
```diff
@@ -19,7 +19,7 @@ contributors: Jordan Harband
 		1. If ? IsRegExp(_regexp_) is *true*, then
 			1. Let _R_ be _regexp_.
 		1. Else,
-			1. Let _R_ be ? RegExpCreate(_regexp_, *undefined*).
+			1. Let _R_ be ? RegExpCreate(_regexp_, *"g"*).
```
Comment (littledan) [inline spec.emu]: Nit: Use backticks for strings.

Comment (ljharb) [reply spec.emu]: ah, thanks

**Pull request merged**